.github/workflows/kind-e2e-tests.yaml

```diff
@@ -66,12 +66,6 @@ jobs:
           kubectl patch configmap -n pipelines-as-code -p "{\"data\":{\"bitbucket-cloud-check-source-ip\": \"false\"}}" \
           --type merge pipelines-as-code
 
-          # Add an extra Repository ID to the token scope to fetch private url
```

Comment:
remind me why you needed to remove that?

Reply:
But now i am thinking lets keep `TestGithubPullRequestRemoteTaskAnnotations` test which does global repository setting test

WDYT ?

Reply:
sounds good,